aws/resource_aws_emr_cluster_test.go

```diff
@@ -3506,17 +3588,50 @@ resource "aws_emr_cluster" "test" {
 
   master_instance_group {
     instance_type = "m4.large"
+    market        = "ON_DEMAND"
   }
 
   core_instance_group {
     instance_type = "m4.large"
+    market        = "ON_DEMAND"
   }
 
   depends_on = ["aws_iam_role_policy_attachment.test", "aws_route_table_association.test"]
 }
 `, rName)
 }
 
+func testAccAWSEmrClusterConfigCoreInstanceGroupMarket(rName, Market string) string {
```

Comment:
Nit: Go variable names should begin with a lowercase (uppercase is generally reserved for exported names) -- this test configuration looks good otherwise 👍 

```suggestion
func testAccAWSEmrClusterConfigCoreInstanceGroupMarket(rName, market string) string {
```